projects/mangrove-monitoring/mangrove-monitoring_updates.md

```diff
@@ -6,4 +6,6 @@ title: Mangrove Monitoring
 category: mangrove-monitoring
 enable_nav: true
 ---
+# REU Summer 2024
 
+During the summer of 2024, Gage Wrye used drone imagery from the 2020 Jamaica expedition to train a UNet classifier to detect the presence of mangroves. [Drone Classification](https://github.com/UCSD-E4E/ml-mangrove/tree/master/DroneClassification)
```

Comment:
@Sean1572 is this the way we want to use the updates pages?  I think this might break something?

Reply:
No, this should be made as a blog post and it will be automatically added to mangrove updates